docs/source/guides/contributor/chapters/plugins.rst

```diff
@@ -183,6 +183,35 @@ has to be executed early in the code so try to keep the required deps
 minimal (for example the `avocado.core.settings.settings` is not yet
 available).
 
+Plugins execution order
+=======================
+Avocado execute plugins serially one after each other. Because of that, 
+there are cases where the order of executions is important. For example, 
+package plugin installs necessary dependencies and sysinfo plugin collects 
+system information to check if the dependencies were installed. To ensure that 
+package plugin will be executed before sysinfo plugin, avocado is introducing 
+plugin priority.
+
+The plugins with higher priority will be executed earlier than the plugins with 
+lower priority. The priorities are on the scale between 1-100 and as default 
+all plugins have priority set to PLUGIN_PRIORITY_NORMAL (50). For easier usage, 
+avocado has predefined values in `avocado.core.extension manager`::
```

Comment:
If all plugins should have this default priority, why you are using `PLUGIN_PRIORITY_VERY_LOW` and `PLUGIN_PRIORITY_HIGH`, for exec-test and instrumented? It was not clear.